app/src/main/java/edu/uc/clemenks/keepingupwithschool/ui/main/MainFragment.kt

```diff
@@ -93,4 +89,7 @@ class MainFragment : Fragment() {
         }
     }
 
+    companion object {
+        fun newInstance() = MainFragment()
+    }
```

Comment:
I moved the companion object based on the suggestion here: https://kotlinlang.org/docs/reference/coding-conventions.html